dev_utils/src/automatic_code_generator/cpp/enumeration_builder/enumeration_builder.py

```diff
@@ -264,13 +280,24 @@ def main():
     # Parse arguments
     args = parse_options()
 
+    # Set log level
+    if args.debug:
+        logging.basicConfig(level=logging.DEBUG)
+
+    # Set output file
+    output_file = (args.output if args.output else f'{args.enum}.hpp')
+
+    logging.debug(f'Command executed {get_command_executed()}')
```

Comment:
NIT: path is not correctly printed when it consists of more than just a file (as in the provided example).